src/app/pages/quickMaps/pages/dietaryChange/components/options/options.component.scss

```diff
@@ -1,19 +1,126 @@
 @import 'src/assets/scss/_colour';
 
 .container {
-  display: flex;
   margin-bottom: 10px;
   border: 1px solid $color_primary;
   border-radius: 5px;
+  align-items: center;
+  background-color: rgb(252, 250, 250);
+  display: flex;
+  width: 100%;
   flex-flow: row wrap;
-  background-color: white;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
+  font-size: 18px;
+  gap: 5px;
+
+  & .wideItem {
```

Comment:
Should use kabab-case in css, rather than camelCase

Reply:
👀  Kebab.... *runs*

Reply:
There's still more of these in the same file. Sorry.